.github/ISSUE_TEMPLATE/--feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "✨Feature request"
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+## Is your feature request related to a problem?
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+## Describe the solution you'd like
+A clear and concise description of what you want to happen.
+
+## Describe alternatives you've considered
+A clear and concise description of any alternative solutions or features you've considered.
+
+## Additional context
```

Comment:
maybe `## Additional context and screenshots`

Reply:
I think the line below covers screenshots, I'll make it a comment like the other files.